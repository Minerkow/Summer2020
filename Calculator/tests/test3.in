asd = ?;
zoo  = asd -529 /(187 -209 )-770 -585 /194 *321 *(832 +427 )/876 ;
zoo  = 386 -zoo /(540 -zoo +820 /(391 *443 )+630 );
zoo  = 673 *((526 *630 ))/423 -zoo +633 *146 -452 /zoo -933 +600 +zoo +390 -zoo *392 -194 *939 ;
dg  = 900 /3 -465 *99 /717 *480 /zoo /660 -69 ;
xq  = 475 -dg ;
print zoo;
print dg;
print xq;

