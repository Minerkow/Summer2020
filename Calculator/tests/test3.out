286227
231
244